forge-game/src/main/java/forge/game/GameView.java

```diff
@@ -200,15 +205,29 @@ void updateGameLog(GameLog gameLog) {
     public TrackableCollection<CardView> getRevealedCollection() {
         return get(TrackableProperty.RevealedCardsCollection);
     }
-
     public void updateRevealedCards(TrackableCollection<CardView> collection) {
         set(TrackableProperty.RevealedCardsCollection, collection);
     }
 
+    public String getDependencies() {
+        return get(TrackableProperty.Dependencies);
+    }
+    public void setDependencies(Table<StaticAbility, StaticAbility, StaticAbilityLayer> dependencies) {
+        StringBuilder sb = new StringBuilder();
+        StaticAbilityLayer layer = null;
+        for (Cell<StaticAbility, StaticAbility, StaticAbilityLayer> dep : dependencies.cellSet()) {
+            if (layer != dep.getValue()) {
+                layer = dep.getValue();
+                sb.append("Layer " + (StaticAbilityLayer.CONTINUOUS_LAYERS.indexOf(layer) + 1)).append(": ");
+            }
```

Comment:
I have questions about this table:
- is it that only two StaticAbility have only a dependency in 1 layer? Or could they be in multiple?
- about the layer variable, you want to use it as prefix, in the description, assuming the table is somewhat ordered by their value, instead of their keys?
- the number doesn't work for the sub layers, like 7c. Would it make sense to add the number to `StaticAbilityLayer` ?

Reply:
1. I think in theory it might be possible to produce, but would probably involve some pretty specific copy steps?
But if you mean I should rather store them as Set to be safe I agree!
2. yea because that's the order layers get processed it made sense to display it the same way
3. ah good point, guess I'll add them there